ocfweb/tv/templatetags/tv_formatting.py

```diff
@@ -0,0 +1,14 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def tv_lab_hours(hour):
+    if hour:
+        if hour.open.minute != 0 or hour.close.minute != 0:
+            fmt = '{:%-I:%M%P}-{:%-I%M%P}'
+        else:
+            fmt = '{:%-I%P}-{:%-I%P}'
+
+        return fmt.format(hour.open, hour.close)
```

Comment:
https://github.com/ocf/ocfweb/blob/0ae729c357a2b1d5002b393fafbc134e8a9b198b/ocfweb/templatetags/lab_hours.py#L14
Similar to this function; maybe try to adapt it? Not really necessary, your call.